android-sdk-restclient-test/src/AndroidManifest.xml

```diff
@@ -0,0 +1,17 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+
+          package="healthvault.microsoft.com.android_sdk_restclient_test"
+>
+
+    <uses-sdk
+        android:minSdkVersion="21"
+        android:targetSdkVersion="25" />
+    <uses-permission android:name="android.permission.INTERNET"></uses-permission>
```

Comment:
nit: same comment as in the other manifest. Make sure things look the same across tags/files.